index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/a70cf7601d.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="./css/styles.css" />
+    <title>Personal Portfolio</title>
+  </head>
+
+  <body>
+    <section>
+      <header class="header">
+        <nav class="nav">
+          <a href="">Welcome</a>
+          <i class="fas fa-bars"></i>
+        </nav>
+      </header>
+      <div class="hero-heading">
+        <h1>Hey there, I am Dennis.</h1>
+        <h1 id="title">I'm a software Developer</h1>
```

Comment:
- [ ] Very nice working including this section in its own element 👍. Although the requirements state that you should have the "I'm a software Developer" text inside a `p` tag. 
something like this would be really great
```suggestion
        <p id="title">I'm a software Developer</p>
```